src/main/kotlin/no/nav/tilleggsstonader/klage/behandling/domain/Behandling.kt

```diff
@@ -35,7 +35,7 @@ data class Behandling(
 )
 
 data class PåklagetVedtakDetaljer(
-    val fagsystemType: FagsystemType,
+    val fagsystemType: FagsystemType?,
```

Comment:
Hvorfor blir denne optional?

Reply:
Midlertidig for å sikre kompatibilitet mellom sak og klage sidan kontrakter er setter denne til optional midlertidig.